### PR TITLE
Signal ping received. Releasing first preparatory modules.

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
Content "Adds first wave of exodii map extras, crashed basic resource pods."

#### Purpose of change
The exodii are a planned faction of interdimensional survivor-traders. One of their early phase transitions to a new world they plan to set up shop in is to scatter resource pods around using very cheap dimensional portals with poor targeting. These pods will be tracked down by exodii scouts sometimes, or can be found and explored by players.

#### Describe the solution
Adds basic exodii pods. For now the resources they contain are in early development, later on there will be some more versions with other possibly useful items, but most of the time the stuff inside is going to be basically garbage.

Occasionally, pods will spawn with either dangerous zombies that caught a ride along, or with exodii scout units that have tracked down and secured the pod.

#### Describe alternatives you've considered
I thought of adding the exodii base first, but this gets them into the game without as much work.

#### Testing
Spawns in game, no errors. Could use more testing as I have changed a few settings since that.

#### Additional context
Maybe someone wants to think of something to do with a 10,000 kg block of pure lead.

Some missing bits in the previously unused exodii monster definitions were noticed by discord, I am updating those.

### CODE REVIEW

**Comment by SenpaiSlime** [comment]
Have you considered adding things like dermatik nests happening inside the pods once they land? there might be a few other fitting creatures that would have some degree of interest in the pods

File: data/json/itemgroups/SUS/exodii.json [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+          { "item": "exodii_ingot_copper", "prob": 50 },
+          { "item": "exodii_ingot_lead", "prob": 30 },
+          { "item": "exodii_ingot_tin", "prob": 10 },
+          { "item": "exodii_ingot_zinc", "prob": 10 }
+        ],
+        "prob": 60
+      }
+    ]
+  }
+]
```
Comment (anothersimulacrum) [inline data/json/itemgroups/SUS/exodii.json:149]:
```suggestion
]

```

File: data/json/mapgen/map_extras/exodii_crashes.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+        "|_|",
+        "|||"
+      ],
+      "palettes": [ "exodii_crash_palette" ],
+      "items": {
+        "_": [ { "item": "SUS_exodii_storage_lowvalue", "chance": 100 } ]
+      }
+    }
+  }
+]
```
Comment (anothersimulacrum) [inline data/json/mapgen/map_extras/exodii_crashes.json:237]:
```suggestion
]

```

**Comment by I-am-Erk** [comment]
> 
> 
> Have you considered adding things like dermatik nests happening inside the pods once they land? there might be a few other fitting creatures that would have some degree of interest in the pods

There's a chance of zomborgs spawning around them sometimes, these pods aren't really big enough for dermatik to nest inside, and seem to be hermetically sealed.

File: data/json/items/resources/alien.json [hunk 3]
```diff
@@ -145,5 +145,96 @@
     "price": 2000,
     "material": [ "copper", "ceramic" ],
     "category": "spare_parts"
+  },
+  {
+    "id": "exodii_ingot_aluminum",
+    "category": "spare_parts",
+    "name": { "str": "Enormous aluminum ingot" },
```
Comment (LaVeyanFiend) [inline data/json/items/resources/alien.json:152]:
```suggestion
    "type": "GENERIC",
    "id": "exodii_ingot_aluminum",
    "category": "spare_parts",
    "name": { "str": "enormous aluminum ingot" },
```
Capitalization & type applies to all ingots

**Comment by I-am-Erk** [comment]
image
Can anyone see why I'm getting this? I can't spot what I did differently from my wilderness map extras here.

**Comment by I-am-Erk** [comment]
image
Getting there. Needs adjusting but getting there.

File: data/json/items/resources/alien.json [hunk 4]
```diff
@@ -145,5 +145,103 @@
...
+    "material": [ "copper" ],
+    "symbol": "#",
+    "color": "light_green"
+  },
+  {
+    "id": "exodii_ingot_zinc",
+    "type": "GENERIC",
+    "category": "spare_parts",
+    "name": { "str": "enormous zinc ingot" },
+    "description": "An absolutely massive block of pure zinc. Bet you didn't expect that... nobody expects zinc.",
```
Comment (actual-nh) [inline data/json/items/resources/alien.json:224]:
```suggestion
    "description": "An absolutely massive block of pure zinc.  Bet you didn't expect that -  nobody expects zinc!",
```

**Comment by actual-nh** [comment]
> There's a chance of zomborgs spawning around them sometimes, these pods aren't really big enough for dermatik to nest inside, and seem to be hermetically sealed.

If an area happened to have a high concentration of the pods, then I can see one or two nether critters slipping through due to the dimensional disruption.

**Comment by PatrikLundell** [comment]
I don't really understand how zombies could come with the pods unless the zombies were exodii originally (and thus ought to end up as new types of zombies), as I assume there are no humans that can be zombiefied in the dimension the exodii sent the pods from.
IF zombies came with the pods for a ride, they ought to remain inside them until the pods are cracked open, unless the exodii both have a terrible perimeter security at their launching sites, and the launchers are some kind of pads where everything on the pad is beamed over.

**Comment by pehamm** [comment]
> I don't really understand how zombies could come with the pods unless the zombies were exodii originally (and thus ought to end up as new types of zombies), as I assume there are no humans that can be zombiefied in the dimension the exodii sent the pods from.
> IF zombies came with the pods for a ride, they ought to remain inside them until the pods are cracked open, unless the exodii both have a terrible perimeter security at their launching sites, and the launchers are some kind of pads where everything on the pad is beamed over.

Zomborgs are non-normal zombies that previously were exodii. The exodii are themselves humans from other dimensions the blob invaded, only that unlike us they got portal technology just far enough to get out. They can still zombify like we do.

The portals they have are not the most precise, and I imagine they will sometimes spit you out in situations where it is tough for the pods to maintain structural integrity, like a few km above ground. A good chunk of the pods will probably be damaged as a result of that.

**Comment by I-am-Erk** [comment]
As far as z-axial stuff is concerned, I've got some fluff about gravity wells and object density that will probably never see the light of day that is meant to explain why things don't appear in mid-air or deep underground very often. It might be fun to have a few of them but I don't want that to be the standard case.

**Comment by faefux** [comment]
Finally! I am so hype!!!!  
Sorry that it is unrelated, but relating to CBM's: Are we going to overhaul CBM's soon with the introduction of Exodii then? Aside from gating them through the new faction will the limits and limitations of them change? 
Do we have a discussion on github about this already?
Want to know all the things! Wish Lore discussion wasn't locked behind Discord. T.T

**Comment by I-am-Erk** [comment]
> Finally! I am so hype!!!!
> Sorry that it is unrelated, but relating to CBM's: Are we going to overhaul CBM's soon with the introduction of Exodii then? Aside from gating them through the new faction will the limits and limitations of them change?
> Do we have a discussion on github about this already?
> Want to know all the things! Wish Lore discussion wasn't locked behind Discord. T.T

The exodii need to be able to sell you and install CBMs before we remove them from the base game.
With the introduction of the faction I will probably update the design document, but the actual game won't reflect that for a while until I can implement all my plans.

Most cbms are unlikely to change much, if at all. Access to them will be different, mostly easier, and we'll have more excuses to have weird CBMs that previously did not fit the lore.

**Comment by kholat** [comment]
bionicle cdda? BIONICLE CDDA!!!!!

**Pull request merged**